### ISSUE TITLE
Setting the maximum number of steps

### ISSUE THREAD

**Issue by Heidisnaps** [thtrieu/darkflow#366]
Hi.

I want to set the number of training steps.
For example, train up to 1000 times and quit. Or end the training if the loss value is less than 10.
Is there an option like this?
